src/components/pages/TopicCreation/Steps/CreationSuccess.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { Button, Tooltip } from 'antd';
+
+const CreationSuccess = props => {
+  const [tooltip, setTooltip] = useState('Copy to Clipboard'); // tooltip text
+  const [joincode, setJoincode] = useState('6xT4D'); // dummy code for now, this is from topic in redux
+  if (props.currentStep !== 6) {
+    return null;
+  }
+
+  // function to copy join code to clipboard
```

Comment:
Really nice!  I'm curious if the behavior of this function would be effected if updated to the new React 17.  I don't think it will, but something to keep an eye on in the future.